src/Console/ConsoleIo.php

```diff
@@ -489,7 +489,7 @@ protected function _getInput(string $prompt, ?string $options, ?string $default)
         $this->_out->write('<question>' . $prompt . "</question>$optionsText\n$defaultText> ", 0);
         $result = $this->_in->read();
 
-        $result = trim($result);
+        $result = $result !== null ? trim($result) : '';
```

Comment:
```suggestion
        $result = $result === null ?  '' : trim($result);
```

Reply:
There's a difference between these two?

Reply:
You avoid an unnecessary negation and cognitive load.